server/src/alerts/target.rs

```diff
@@ -325,20 +327,34 @@ impl CallableTarget for AlertManager {
           }
         }]);
 
+        let alert = &mut alerts[0];
+
+        alert["labels"].as_object_mut().expect("is object").extend(
```

Comment:
Small suggestion, we should not have all the rules details as the standard labels (additional_labels) as they will be visible and if just one changes it create a new alert.

We could have the more internal (metadata) labels (__meta_xx), for these kind of rule details

Reply:
Thanks @syepes . It is not clear from the docs, but are the Prometheus relabeling rules applicable for alertmanager as well?

Reply:
> Thanks @syepes . It is not clear from the docs, but are the Prometheus relabeling rules applicable for alertmanager as well?

https://github.com/parseablehq/parseable/issues/311#issuecomment-1446523836